kontiki.cabal

```diff
@@ -21,32 +21,33 @@ Source-Repository head
 
 Library
   Exposed-Modules:     Network.Kontiki.Raft
+                     , Data.Kontiki.MemLog
```

Comment:
I'm not very fond of this. The `MemLog` implementation is something that can be useful in tests, and may be useful in some demo tools, but it defies all guarantees a system like Raft needs from its log implementation w.r.t. (crash) safety.

As such, I don't think this module should be part of the library and definitely not be exposed as part of its API to any users.